server/src/internalClusterTest/java/org/opensearch/snapshots/SegmentReplicationSnapshotIT.java

```diff
@@ -100,11 +100,11 @@ public void ingestData(int docCount, String indexName) throws Exception {
     // Start cluster with provided settings and return the node names as list
     public List<String> startClusterWithSettings(Settings indexSettings, int replicaCount) throws Exception {
         // Start primary
-        final String primaryNode = internalCluster().startNode();
+        final String primaryNode = internalCluster().startNode(featureFlagSettings());
```

Comment:
Sorry @tan31989 , I missed these changes earlier.
1. Need to resume this `assumeFeatureFlag ` method call [here](https://github.com/opensearch-project/OpenSearch/pull/4959/files#diff-4ca892b67e6bb252096508fccda6dcaa9b3acdf1c24b4b42f47a3d25b19deddbR50-R52)
2. Need to enable feature flag for Segment replication similar to what's done in SegmentReplicationIT.java

Reply:
@dreamer-89 Can we do this as a follow up?

Reply:
I have addressed that [here](https://github.com/opensearch-project/OpenSearch/pull/4959/files#diff-befd32a159f6d02a5c753c4226800355efcbdbb6c126e761c9eae1d04d60ff59L73-R98). Please do let me know if anything more needs to be addressed, happy to incorporate that in this PR.

Reply:
Thanks @tan31989 for accomdodating changes here.